src/SDKs/Search/DataPlane/Search.Tests/Properties/AssemblyInfo.cs

```diff
@@ -3,8 +3,8 @@
 // license information.
 
 using System.Reflection;
-using System.Runtime.CompilerServices;
 using System.Runtime.InteropServices;
+using Xunit;
```

Comment:
@brjohnstmsft any reason this is required in AssemblyInfo.cs? If not please remove.